### ISSUE TITLE
javascripting - I can't use arrows in the menu to select next lesson

### ISSUE THREAD

**Issue by dr-w9999** [nodeschool/discussions#1565]
I did 3 lessons in javascripting.
When I run "javascripting" again arrows don't work and I can't select next lesson.

**Comment by dr-w9999** [comment]
I closed "cmd" console and started again, but still it didn't work.
I checked in an editor and the arrows work there.
Only now after I singed out from windows and login again it's working as expected.

**Comment by bdharris08** [comment]
If anyone else has this problem, you can use <code>>expressworks print lessonName </code>

**Comment by RaeRae961** [comment]
I have been having this same problem with the scope-chains module. (I couldn't arrow through javascripting, but it did put me at the next lesson automatically after I completed the previous one.) I have tried logging in and out, and also the suggestion to use >expressworks print lessonName. I have tried various command lines, with either Git CMD or the regular Windows command line, and they both come back with: 'expressworks' is not recognized as an internal or external command, operable program or batch file.
I cannot move past the first lesson in this module at all.
